pyproject.toml

```diff
@@ -15,6 +15,7 @@ opentimelineio = "^0.16.0"
 pytest = "^6.2.0"
 opencolorio = "^2.3.2"
 selenium = "^4.20.0"
+sphinx = "^7.3.7"
```

Comment:
I could not resolve this with on windows with my poetry lock, but this is not a problem since once I had removed it it had been recreated again correctly.